front/app/api/user_custom_fields/useUserCustomField.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import userCustomFieldssKeys from './keys';
+import { IUserCustomField, UserCustomFieldsKeys } from './types';
+
+const fetch = ({ customFieldId }: { customFieldId: string }) =>
```

Comment:
Minor: we could simplify to something like
```suggestion
const fetch = (customFieldId: string ) =>
```